test/libsolidity/syntaxTests/pragma/invalid_range_operator_range_second.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity 0.8.17 - >0.8.20;
```

Comment:
Please also add a case without whitespace or with shorter versions. This sometimes makes a difference. For example `0.0 - =1.0` was valid, while `0.0-=1.0` was not. We should make sure that both cases are handled.